src/client/styles/components/Feedback.scss

```diff
@@ -0,0 +1,136 @@
+.feedback {
+    position: static;
+    bottom: 0;
+    right: 0;
+    z-index: 100;
+    text-align: right;
+    background: #54514A;
+    color: #000;
+  
+    @include min-screen(1024px) {
```

Comment:
nit: I agree it's probably not necessary to have the button in mobile, but is 1024 too big of a min-screen?

Reply:
This pins the form to the bottom of the window and forces it full width. The original code I borrowed this from has this as a variable "tablet-portrait" and 1024 is the best estimate of that value I could come up with.

In testing it out it seems to work well in Chrome at least.